<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingLoadReleaseDialog</class>
 <widget class="QDialog" name="SettingLoadReleaseDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>332</width>
    <height>536</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>311</width>
     <height>411</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Mng</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Algorithm</string>
    </property>
   </column>
  </widget>
  <widget class="QPushButton" name="pushButtonOK">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>490</y>
     <width>112</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>OK</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxMasterBuff">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Master image buffer</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="checkBoxTargetBuff">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>291</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Target image buffer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
